x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/home_page.ts

```diff
@@ -11,87 +11,105 @@ import { getTestAlertData, getTestActionData } from '../../lib/get_test_data';
 
 export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const testSubjects = getService('testSubjects');
+  const security = getService('security');
   const pageObjects = getPageObjects(['common', 'triggersActionsUI', 'header']);
   const log = getService('log');
   const browser = getService('browser');
   const supertest = getService('supertest');
   const objectRemover = new ObjectRemover(supertest);
 
   describe('Home page', function () {
-    before(async () => {
-      await pageObjects.common.navigateToApp('triggersActions');
-    });
+    describe('Loads the app with limited privileges', () => {
```

Comment:
This is the new test added to test for this scenario. It switches to using the `alerts_and_actions_role` to make sure the page loads.

Reply:
This is great! Thanks for a quick turn around!  We could add a follow up PR for the tests  to be run as a `test_user` with the right set of roles and privileges instead of `all`

Reply:
LGTM